source/schema/schema-management-upgrade.txt

```diff
@@ -0,0 +1,206 @@
+=========================
+Schema Management Upgrade
```

Comment:
It might be better to include v2.11 somewhere in the title so users will know if this applies to them.

Also seeking feedback on this title and where it should live in the ToC

Reply:
I think the ToC location is correct, but I would add some words in the first paragraph to make it clear that all this only applies to users who were using an earlier BI Connector version and are now using 2.11 or later. Also we should be sure to link here in the release notes for 2.11.

At a high level, I'm worried that using "Upgrade" to mean "changing the way you use command line options" may be confusing to some users. An upgrade is something you do once, but using the new command line options is something you do every time you use the tool. If possible I'd remove "upgrade path" from the lower sections of the page and make it clear that "upgrade" refers only to moving from version 2.10 or earlier to version 2.11 or later. You could use "old behavior/new behavior" or "old usage/new usage" or something to refer to the differences in command line options.

Reply:
- Changed the title.
- Added a link to this page from the release notes.
- Changed or removed all mentions of "upgrade"